README.md

```diff
@@ -1,5 +1,47 @@
 # dcache
 
+
+[![PyPI](https://img.shields.io/pypi/v/dcache.svg)](https://pypi.python.org/pypi/dcache)
+
+[![Coverage Status](https://coveralls.io/repos/github/HBN3tw0rk/dcache/badge.svg?branch=master)](https://coveralls.io/github/HBN3tw0rk/dcache?branch=master)
+
+[![Documentation Status](https://readthedocs.org/projects/dcache/badge/?version=latest)](https://dcache.readthedocs.io/en/latest/?version=latest)
+
+
+
+Distributed Cache for Humans
+
+
+- Documentation: https://dcache.readthedocs.io.
+
+
+## Installation
+
+```bash
+pip install dcache
+```
+
+
+
+## How to Use
+
+
+## Contributing
+Contributions are welcome, feel free to open an Issue or Pull Request.
+
+Pull requests must be for the `develop` branch.
+
+```bash
+git clone https://github.com/HBN3tw0rk/dcache
+cd dcache
+git checkout develop
+python -m venv .venv
+pip install -r requirements-dev.txt
```

Comment:
Fix the filename to `requirements_dev.txt`

Reply:
```suggestion
pip install -r requirements_dev.txt
```